pootle/static/js/editor/containers/Editor.js

```diff
@@ -11,7 +11,7 @@ import React from 'react';
 import EditingArea from '../components/EditingArea';
 import RawFontTextarea from '../components/RawFontTextarea';
 import { getAreaId } from '../utils';
-
```

Comment:
Rather place the import below before the block above (project-wide imports should come first, then module-relative imports). I was under the impression there was an ESLint rule for this though...